lib/election.js

```diff
@@ -732,12 +732,15 @@ function _verifyVote(ledgerNode, vote, electionTopic, callback) {
 }
 
 function _verifyVoteSignature(vote, callback) {
-  // TODO: optimize
   jsigs.verify(vote, {}, (err, result) => {
     if(err) {
       return callback(err);
     }
-    // TODO: check result
+    if(!result.verified) {
+      return callback(new BedrockError(
+        'Vote signature could not be verified.',
+        'OperationError', {vote, keyResults: result.keyResults}));
```

Comment:
Hmm, fine other than `OperationError`. I don't think this is an operation error, more like a security error or something. Will think on it more and get back to you.

Reply:
Error text should be:

`Vote signature verification failed.`

With error type:

`AuthenticationError`.